rllib/algorithms/bc/bc.py

```diff
@@ -116,11 +115,6 @@ def get_default_learner_class(self) -> Union[Type[Learner], str]:
 
     @override(MARWILConfig)
     def validate(self) -> None:
-        # Can not use Tf with learner api.
```

Comment:
This is already checked in `validate()`. We should never(!) automatically change properties inside AlgorithmConfig (unless private ones that are covered by (public) @properties).